app/services/api/v1x1/group.rb

```diff
@@ -0,0 +1,7 @@
+module Api
+  module V1x1
+    module Group
+
```

Comment:
Shouldn't we have class Seed < Api::V1x0::Group::Seed; end here?

Reply:
Same as https://github.com/RedHatInsights/catalog-api/pull/648#discussion_r410467046